scripts/parse_data_from_cldr.py

```diff
@@ -0,0 +1,143 @@
+"""
```

Comment:
This script is not parsing data from CLDR only, but also adding the "supplementary data", so I think that the name of the file should be changed. You can use the same than in `dateparser` ("write_complete_data") or find another (better) name.